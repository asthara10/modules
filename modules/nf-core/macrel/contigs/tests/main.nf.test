nextflow_process {

    name "Test Process MACREL_CONTIGS"
    script "../main.nf"
    process "MACREL_CONTIGS"

    tag "modules"
    tag "modules_nfcore"
    tag "macrel"
    tag "macrel/contigs"

    test("test_macrel_contigs") {


        when {
            process {
                """
                
                input[0] = [ 
        [ id:'test', single_end:false ], // meta map
        file(params.test_data['bacteroides_fragilis']['illumina']['test1_contigs_fa_gz'], checkIfExists: true) 
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
