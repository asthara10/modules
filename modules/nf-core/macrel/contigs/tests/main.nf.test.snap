{
    "versions": {
        "content": [
            [
                "versions.yml:md5,6783c35d2069f446d491d6ecb76cef9a"
            ]
        ],
        "timestamp": "2023-11-24T10:36:01.399157"
    }
}